doc/api/assert.md

```diff
@@ -46,11 +59,17 @@ and:
   [`assert.strictEqual()`] is used.
 * `expected` {any} Set to the expected value in case e.g.,
   [`assert.strictEqual()`] is used.
-* `generatedMessage` {boolean} Indicates if the message was auto-generated
-  (`true`) or not.
 * `code` {string} This is always set to the string `ERR_ASSERTION` to indicate
   that the error is actually an assertion error.
+* `generatedMessage` {boolean} Deprecated. Indicates if the message was
+  auto-generated (`true`) or not.
 * `operator` {string} Set to the passed in operator value.
+* `userMessage` {any} Contains the actual passed through message by the user.
+  It will not be set in case the user does not provide a error message.
+
+All error messages thrown by the `assert` module are auto-generated. If the user
```

Comment:
This sentence is confusing to me. I don't think `auto-generated` is a clear term and should probably be avoided.

@nodejs/documentation